test/integration/default/controls/remote_pkgs_spec.rb

```diff
@@ -1,4 +1,11 @@
 ### REMOTE PKGS
+remote_pkgs = ['zoom']
```

Comment:
Not very Rubyish ^^

```ruby
remote_pkgs = 
  case os[:name]
  when 'opensuse'
    []
  else
    ['zoom']
  end
```

Reply:
okay for this. I just don't like `%w()` :)